calico_rkt/calico_rkt.py

```diff
@@ -146,15 +152,20 @@ def execute(self):
             _log.debug("Starting plugin execution.")
             self._execute()
         except SystemExit, e:
-            # SystemExit indicates an error that was handled earlier
-            # in the stack.  Just set the return code.
-            rc = e.code 
+            # Defer printing the error message and setting the rc in the
+            # finally clause
+            pass
         except BaseException:
             # An unexpected Exception has bubbled up - catch it and
             # log it out.
-            _log.exception("Unhandled Exception killed plugin")
```

Comment:
We still want this _log.exception here - it will include the stacktrace which killed the plugin, which is crucial for debugging.